src/components/layout/NodeInfo.tsx

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios'
+import * as React from 'react'
+
+const enum Environments {
+  kauTestnet = 'kau-testnet',
+  kagTestnet = 'kag-testnet',
+  kauMainnet = 'kau-mainnet',
+  kagMainnet = 'kag-mainnet',
+}
+
+const REGION_ERROR = {error: 'Region Offline'}
+
+const MONITOR_ENDPOINTS = {
+  [Environments.kauTestnet]: [
+    'http://kau-testnet-oceania.kinesisgroup.io:4000',
+    'http://kau-testnet-asia.kinesisgroup.io:4000',
+    'http://kau-testnet-america.kinesisgroup.io:4000',
+    'http://kau-testnet-europe.kinesisgroup.io:4000',
+  ],
+  [Environments.kagTestnet]: [
+    'http://kag-testnet-oceania.kinesisgroup.io:4000',
+    'http://kag-testnet-asia.kinesisgroup.io:4000',
+    'http://kag-testnet-america.kinesisgroup.io:4000',
+    'http://kag-testnet-europe.kinesisgroup.io:4000',
+  ],
+}
+
+function getInfo(ep: string) {
+  return axios.get(ep)
+    .then((d) => d.data)
+    .catch(() => (REGION_ERROR))
+}
+
+async function loadData() {
+  const data = await Promise.all(Object.entries(MONITOR_ENDPOINTS).map(async ([environment, endpoints]) => {
+    const envInfo = await Promise.all(endpoints.map(async (ep) => {
+      const info = await getInfo(ep)
+      return {[ep]: info}
+    }))
+
+    const mergedEnvInfo = envInfo.reduce((acc, val) => ({...acc, ...val}), {})
+    return {[environment]: mergedEnvInfo}
+  }))
+
+  return data.reduce((acc, val) => ({...acc, ...val}), {})
+}
+
+export default class NodeInfo extends React.Component<any, {nodeInfo: any, interval?: any}> {
+  state = {nodeInfo: {}, interval: undefined}
+
+  async componentDidMount() {
+    const setDataOnState = async () => {
+      const nodeInfo = await loadData()
+      this.setState({nodeInfo})
+    }
+
+    const interval = setInterval(setDataOnState, 120000)
+    this.setState({interval})
```

Comment:
Actually better practice to not put the interval in the state, but just as a class variable.